yarn.lock

```diff
@@ -1391,6 +1391,11 @@ acorn-globals@^4.1.0:
     acorn "^6.0.1"
     acorn-walk "^6.0.1"
 
+acorn-jsx@^5.1.0:
```

Comment:
Why changes to this file?

Reply:
This got introduced by `yarn install`, however, this is not related to PR. So I just reverted the changes for this file. Please help to take another look.